packages/core/src/components/tooltip/tooltip.scss

```
@@ -0,0 +1,79 @@
+.atom-tooltip {
+  --arrow-width: 10px;
+  --arrow-height: 8px;
+
+  background: var(--color-neutral-light-1);
+  border-radius: var(--border-radius);
+  color: var(--color-neutral-white);
+  font: var(--text-body-small);
+  letter-spacing: var(--text-body-small-letter);
+  padding: var(--spacing-xsmall);
+  visibility: hidden;
+
+  &__arrow {
+    background: inherit;
+    height: var(--arrow-height);
+    position: absolute;
+    visibility: hidden;
+    width: var(--arrow-width);
+
+    &::before {
+      background: inherit;
+      content: '';
+      height: var(--arrow-height);
+      position: absolute;
+      transform: rotate(45deg);
+      width: var(--arrow-width);
+    }
+  }
+
+  &.open {
+    visibility: visible;
+  }
+
+  &.open > &__arrow {
+    &::before {
+      visibility: visible;
+    }
+  }
+
+  &[data-popper-placement^='top'] > &__arrow {
+    bottom: -3px;
+  }
+
+  &[data-popper-placement^='bottom'] > &__arrow {
+    top: -3px;
+  }
+
+  &[data-popper-placement^='left'] > &__arrow {
+    right: -3px;
+  }
+
+  &[data-popper-placement^='right'] > &__arrow {
+    left: -3px;
+  }
+
+  &__content {
+    align-items: center;
+    display: flex;
+
+    button {
+      display: none;
+    }
+
+    @media (max-width: 768px) {
```

(block) we need to use `rupture` for it

```suggestion
    @include below(mobile) {
```

---

I tried using it but it didn't work, and I thought the utils were automatically imported. 
Importing resolved the problem. [46693eb](https://github.com/juntossomosmais/atomium/pull/367/commits/46693eba961f2aa486f22778890da1f7563b8950)